docs/howto/access-data.md

```diff
@@ -0,0 +1,45 @@
+### **Accessing the data**
+
+**Option 1: Using the data links**
+
+If we already have a workflow in place for downloading our data, we can use *earthaccess* as a search-only library and get HTTP links from our query results. This could be the case if our current workflow uses a different language and we only need the links as input.
+
+```python
+
+# if the data set is cloud hosted there will be S3 links available. The access parameter accepts "direct" or "external", direct access is only possible if you are in the us-west-2 region in the cloud.
+data_links = [granule.data_links(access="direct") for granule in results]
+
+# or if the data is an on-prem dataset
+data_links = [granule.data_links(access="external") for granule in results]
+
+```
+
+> Note: *earthaccess* can get S3 credentials for us, or auhenticated HTTP sessions in case we want to use them with a different library.
```

Comment:
```suggestion
> Note: *earthaccess* can get S3 credentials for us, or authenticated HTTP sessions in case we want to use them with a different library.
```

Reply:
Good catch! We need to set up a codespell pre-commit hook ;)